lib/enumFactory.js

```diff
@@ -23,26 +23,26 @@ var EnumFactory = function() {
         configurable: false,
         enumerable: true,
         value: arg,
-        writable: false
+        writable: false,
       });
       dxList.push(arg);
     }
     Object.defineProperty(Enum, '_values', {
       configurable: false,
       enumerable: false,
       value: dxList,
-      writable: false
+      writable: false,
     });
     Object.defineProperty(Enum, '_string', {
       configurable: false,
       enumerable: false,
       value: stringified(Enum),
-      writable: false
+      writable: false,
     });
     Object.freeze(Enum);
     return Enum;
   } else {
-    throw "No arguments - could not create Enum.";
+    throw 'No arguments - could not create Enum.';
```

Comment:
we should change this to `throw new Error('No arguments - count not create Enum.'`. Please commit a separate PR for this.